routing/router.go

```diff
@@ -4,10 +4,10 @@ import (
 	"net/http"
 
 	"github.com/gorilla/mux"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
-
 	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/handlers"
 	"github.com/kiali/kiali/prometheus/internalmetrics"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
```

Comment:
nitpick - it was correct before. third-party imports are in the second group above - this prometheus import line should be reverted back to where it was.